lighthouse-core/report/scripts/lighthouse-report.js

```diff
@@ -45,7 +45,32 @@ function sendJSONReport() {
   const popup = window.open(VIEWER_URL, '_blank');
 }
 
+/**
+ * Sets up listeners to expand audit `<details>` when the user prints the page.
```

Comment:
this is kind of dumb, but maybe add a clause here that expanding `<details>` isn't possible via a print stylesheet? Whenever I think about this issue my first thought is always, "wait, why can't we just solve this in the print stylesheet?" and it probably will be again when I come across this function in the future :)